extensions/wikia/PowerUser/maintenance/PowerUserAddFrequentMaintenance.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Maintenance script for adding poweruser_frequent property
+ * for users that have made more than 140 edits in the last 60 days.
+ *
+ * @package MediaWiki
+ * @addtopackage maintenance
+ *
+ * @author Adam Karmiński <adamk@wikia-inc.com>
+ *
+ *
+ */
+
+require_once( __DIR__.'/../../../../maintenance/Maintenance.php' );
+
+use Wikia\PowerUser\PowerUser;
+
+class PowerUserAddFrequentMaintenance extends Maintenance {
+	const PERIOD_INTERVAL = '-60 days';
+
+	/**
+	 * Workflow:
+	 * 1. Get potential new PowerUsers of a type frequent
+	 *    (min. of edits and a poweruser property set to false)
+	 * 2. Check each one against statsdb/rollup_wiki_user_events
+	 * 3. Add the property to the new frequent PUs
+	 */
+	public function execute() {
+		$aPotentialPowerUsersIds = $this->getPotentialNewFrequentPowerUsers();
+		foreach ( $aPotentialPowerUsersIds as $iUserId ) {
+			if ( $this->isNewFrequentPowerUser( $iUserId ) ) {
+				$oPowerUser = new PowerUser( User::newFromId( $iUserId ) );
+				$oPowerUser->addPowerUserProperty( PowerUser::TYPE_FREQUENT );
+			}
+		}
+	}
+
+	/**
+	 * Gets an array of IDs of potential new frequebt PowerUsers
+	 * who has made a minimum of frequent edits overall and
+	 * are not PUs yet
+	 *
+	 * @return Array An array of potential PUs IDs
+	 */
+	private function getPotentialNewFrequentPowerUsers() {
+		global $wgExternalSharedDB;
+		$oDB = wfGetDB( DB_SLAVE, [], $wgExternalSharedDB );
+
+		$oSubQuery = ( new WikiaSQL() )
+			->SELECT( 'up_user' )
+			->FROM( 'user_properties' )
+			->WHERE( 'up_property' )->EQUAL_TO( PowerUser::TYPE_FREQUENT )
+			->AND_( 'up_value' )->EQUAL_TO( '1' );
+
+		$aPotentialPowerUsersIds = ( new WikiaSQL() )
+			->SELECT( 'user_id' )
+			->FROM( 'user' )
+			->WHERE( 'user_editcount' )->GREATER_THAN_OR_EQUAL( PowerUser::MIN_FREQUENT_EDITS )
+			->AND_( null )
+				->NOT_EXISTS( $oSubQuery )
```

Comment:
I think this would be better on the same line as the `AND_` above just so the "and not exists" is together on the same line, which matches the other queries with `AND_` and `EQUAL_TO` and so forth.